addons/mail/static/tests/message/message_reply.test.js

```diff
@@ -120,3 +121,36 @@ test("click on message in reply highlights original message", async () => {
     );
     await contains(".o-mail-Message.o-highlighted:contains('This message has been removed')");
 });
+
+test("reply action functionality in Chatter", async () => {
+    const pyEnv = await startServer();
+    const partnerBId = pyEnv["res.partner"].create({ name: "Partner B" });
+    pyEnv["mail.message"].create([
+        {
+            author_id: partnerBId,
+            body: "Test message from B",
+            model: "res.partner",
+            res_id: serverState.partnerId,
+            subtype_id: pyEnv["mail.message.subtype"].search([
+                ["subtype_xmlid", "=", "mail.mt_note"],
+            ])[0],
+        },
+        {
+            author_id: serverState.partnerId,
+            body: "Another msg",
+            model: "res.partner",
+            res_id: serverState.partnerId,
+            subtype_id: pyEnv["mail.message.subtype"].search([
+                ["subtype_xmlid", "=", "mail.mt_note"],
+            ])[0],
+        },
+    ]);
+    await start();
+    await openFormView("res.partner", serverState.partnerId);
+    await click(".o-mail-Message:contains('Test message from B') button[title='Reply']");
+    await contains("button.active", { text: "Log note" });
+    await contains(".o-mail-Composer.o-focused .o-mail-Composer-input", { value: "@Partner B " });
+    await click(".o-mail-Composer button", { text: "Log" });
+    await contains(".o-mail-Message a.o_mail_redirect", { text: "@Partner B" });
+    await contains(".o-mail-Message:contains('Another msg') button[title='Reply']", { count: 0 });
```

Comment:
What's the purpose of asserting `Reply` action is not present on 'Another msg' self message?

The reply feature works also on self messages, it's just that this action has lower precedence over "Edit" and thus is in dropdown. There are already test coverage for that, I don't know what's the use for this here? I would drop this assertion if it has no real use.